services/MerchStoreService.ts

```diff
@@ -291,6 +291,68 @@ export default class MerchStoreService {
       await user.reload();
       const merchItemOptionRepository = Repositories.merchStoreItemOption(txn);
       const itemOptions = await merchItemOptionRepository.batchFindByUuid(originalOrder.map((oi) => oi.option));
+
+      const totalCost = await this.verifyOrder(originalOrder, user);
+
+      const merchOrderRepository = Repositories.merchOrder(txn);
```

Comment:
The naming of `verifyOrder` makes it slightly confusing as to why a number is returned from the method, since verification implies that an input is either valid or not. So it might make sense to rename this method to `verifyOrderAndReturnTotalCost`, but that might be a bit overkill. @sumeet-bansal any thoughts on this? We could also have a separate method `calculateTotalCostOfOrder` or something similar and use that within the verify method and the placeOrder method. Or just document what the return value means in a doc comment.